path.go

```diff
@@ -10,16 +10,40 @@
 package unison
 
 import (
+	"fmt"
 	"runtime"
 
+	"github.com/lafriks/go-svg"
 	"github.com/richardwilkes/toolbox/errs"
+	"github.com/richardwilkes/toolbox/xmath/geom"
 	"github.com/richardwilkes/unison/enums/arcsize"
 	"github.com/richardwilkes/unison/enums/direction"
 	"github.com/richardwilkes/unison/enums/filltype"
+	"github.com/richardwilkes/unison/enums/paintstyle"
 	"github.com/richardwilkes/unison/enums/pathop"
+	"github.com/richardwilkes/unison/enums/strokecap"
+	"github.com/richardwilkes/unison/enums/strokejoin"
 	"github.com/richardwilkes/unison/internal/skia"
 )
 
+var strokeCaps = map[svg.CapMode]strokecap.Enum{
+	svg.NilCap:       strokecap.Butt, // default to something that won't break
+	svg.ButtCap:      strokecap.Butt,
+	svg.SquareCap:    strokecap.Square,
+	svg.RoundCap:     strokecap.Round,
+	svg.CubicCap:     strokecap.Butt, // default to something that won't break
```

Comment:
Alternatively the unsupported cap modes could be removed from the map. And the `createStrokePaint` method would be changed to return an error if there is no entry in the map for a mode that is encountered in a path.

And the same for unsupported join modes in the `strokeJoins` map below.

Reply:
If an unsupported cap mode or join mode is encountered an error is now returned from path construction.